README.md

```diff
@@ -1,4 +1,4 @@
-ST2-ComputerCraft-Package
+ComputerCraft-Support
```

Comment:
Renaming the package ComputerCraft Support is misleding. I agree about removing the "ST2" part, but "Support" is questionable. I would prefer "ComputerCraft Package" instead.